go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
+
+go 1.23.4
+
+require (
+	github.com/bep/clocks v0.5.0 // indirect
+	github.com/bep/godartsass/v2 v2.3.2 // indirect
+	github.com/bep/golibsass v1.2.0 // indirect
+	github.com/bep/logg v0.4.0 // indirect
+	github.com/caarlos0/env v3.5.0+incompatible // indirect
```

Comment:
For libraries like `github.com/dgrijalva/jwt-go`, consider switching to its maintained fork, `github.com/golang-jwt/jwt`, since the original is not actively maintained.